test/e2e/delete_test.go

```diff
@@ -66,6 +67,10 @@ func testWindowsNodeDeletion(t *testing.T) {
 	}
 
 	// Cleanup secrets created by us.
-	err = framework.Global.KubeClient.CoreV1().Secrets("openshift-machine-api").Delete(context.TODO(), "windows-user-data", meta.DeleteOptions{})
+	err = framework.Global.KubeClient.CoreV1().Secrets(testCtx.namespace).Delete(context.TODO(), "windows-user-data", meta.DeleteOptions{})
 	require.NoError(t, err, "could not delete userData secret")
+
+	err = framework.Global.KubeClient.CoreV1().Secrets(testCtx.namespace).Delete(context.TODO(), secrets.PrivateKeySecret, meta.DeleteOptions{})
```

Comment:
testCtx.namespace would be the tests created to deploy pods, not the machine-api namespace. the private key wouldnt be in there.